templates/meta_woz/dialogues/templates.yaml

```diff
@@ -0,0 +1,111 @@
+dataset: meta_woz
+subset: dialogues
+templates:
+  4b64c6e9-0aa0-431f-85b5-8367daa4773a: !Template
+    id: 4b64c6e9-0aa0-431f-85b5-8367daa4773a
+    jinja: "What does this conversation between a chatbot and a client talk about\
+      \ ? \n{{turns | join(',  ')}}\n|||\n{{domain.replace('_', ' ') | lower }}"
+    name: predict_domain_2
+    reference: ''
+    task_template: false
+  56151136-82a4-455b-98ed-aea6ee1c273d: !Template
+    id: 56151136-82a4-455b-98ed-aea6ee1c273d
+    jinja: '{% set count = namespace(value=0) %}
+
+      {% for i in range(range(turns|length) | random() - 1) %}
+
+      {{["AI Assistant", "Client"][i% 2]}}: {{turns[i]}}
+
+      {% set count.value= i + 1 %}
+
+      {% endfor %}
+
+      {{["AI Assistant", "Client"][ count.value % 2]}}:
+
+      |||
+
+      {{turns[count.value] }}
+
+
+      '
+    name: predict_random
+    reference: ''
+    task_template: true
+  5e90f705-9d63-4917-acc9-3baabc6ee5e9: !Template
+    id: 5e90f705-9d63-4917-acc9-3baabc6ee5e9
+    jinja: "{% set rand_index= namespace(value=range(turns|length)|random()) %}\n\
+      {% for utterance in turns %}\n{% if loop.index0 == rand_index.value %}\n{{[\"\
+      Chatbot\", \"Human\"][loop.index0 % 2]}}: \n{% else %}\n{{[\"Chatbot\", \"Human\"\
+      ][loop.index0 % 2]}}: {{utterance}}\n{% endif %}\n{% endfor %}\n|||\n{{turns[rand_index.value]\
+      \ }}"
+    name: 'predict_missing '
```

Comment:
It seems a little hard for the model to know you want it to fill in this blank. Can you put something like `(blank)` in place of the missing utterance, and then at the end of the prompt ask the model to fill in the blank?